drivers/espi/espi_it8xxx2.c

```diff
@@ -724,18 +724,23 @@ static int espi_it8xxx2_receive_vwire(const struct device *dev,
 		(struct espi_vw_regs *)config->base_espi_vw;
 	uint8_t vw_index = vw_channel_list[signal].vw_index;
 	uint8_t level_mask = vw_channel_list[signal].level_mask;
-	uint8_t valid_mask = vw_channel_list[signal].valid_mask;
 
 	if (signal > ARRAY_SIZE(vw_channel_list)) {
 		return -EIO;
 	}
 
+#if defined(CONFIG_ESPI_VWIRE_VALID_BIT_CHECK)
+	uint8_t valid_mask = vw_channel_list[signal].valid_mask;
+
 	if (vw_reg->VW_INDEX[vw_index] & valid_mask) {
 		*level = !!(vw_reg->VW_INDEX[vw_index] & level_mask);
 	} else {
 		/* Not valid */
 		*level = 0;
 	}
+#else
+	*level = !!(vw_reg->VW_INDEX[vw_index] & level_mask);
+#endif
```

Comment:
Style nit, something like this is cleaner than #ifery for simple changes: 

```c
*level = 0;
if (IS_ENABLED(CONFIG_ESPI_VWIRE_VALID_BIT_CHECK) &&
    && vw_reg->VW_INDEX[vw_index] & valid_mask) {
	*level = !!(vw_reg->VW_INDEX[vw_index] & level_mask);
}
```

Reply:
looks like i need to use this way
        if (IS_ENABLED(CONFIG_ESPI_VWIRE_VALID_BIT_CHECK)) {
                if (vw_reg->VW_INDEX[vw_index] & valid_mask) {
                        *level = !!(vw_reg->VW_INDEX[vw_index] & level_mask);
                } else {
                        *level = 0;
                }
        } else {
                *level = !!(vw_reg->VW_INDEX[vw_index] & level_mask);
        }